src/js/facility-locator/containers/FacilityDetail.jsx

```diff
@@ -36,13 +36,24 @@ class FacilityDetail extends Component {
 
   renderFacilityInfo() {
     const { facility } = this.props;
-    const { name } = facility.attributes;
+    const { name, facility_type: facilityType } = facility.attributes;
```

Comment:
I believe we can tell the API to send data in camelCase by passing the `X-Key-Inflection` header.